src/main/java/TARS/command/CommandType.java

```diff
@@ -0,0 +1,14 @@
+package TARS.command;
+
+public enum CommandType {
+    BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE;
+
+    public static CommandType fromString(String command) throws IllegalArgumentException {
+        try {
+            return CommandType.valueOf(command.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            // Modifying exception message

```

Comment:
This comment does not sound like it is describing what is happening. You might have to be clearer.